### PR TITLE
Add zelerius as a config option

### PR DESCRIPTION
**Pull request by zelerius**
**XMR-stak feature request for Zelerius Network**

Zelerius Network is switching from CN variant 1 (V7) to CN variant 2 (V8) in order to fight against centralized mining caused by ASIC devices. The issue has been addressed before in the following article written by the Zelerius Network team and published in Medium: https://medium.com/@zelerius/crytonight-variant-2-v8-for-zelerius-network-7ba97489e08e/

The algorithm change is proposed for block 265000, which with an average of 1 block per 30 seconds thrown into the net, it will be accomplished on December 15, 2018. **The hard fork to CN variant 2 (V8) is configured in the Block Major Version 6.**

We request this PR in order to add Zelerius as a config option and automatic switch with block version 6 to CN variant 2 (V8).

**Technical details**

Zelerius Network has algorithm CN variant 1 integrated. It will switch to algorithm CN variant 2 through a Hardfork in the block mentioned previously.

**Blocks version**

```
- Block Major Version 5. Block Minor Version 2 - CN Variant 1
- Block Major Version 6. Block Minor Version 2 - CN Variant 2 - Hard fork
```

We open the PR against **dev** branch as indicated.

Thank you,
Zelerius Developers Team

### CODE REVIEW

File: xmrstak/jconf.cpp [hunk 1]
```diff
@@ -109,7 +109,8 @@ xmrstak::coin_selection coins[] = {
 	{ "qrl",             	 {cryptonight_monero_v8, cryptonight_monero, 255u},   {cryptonight_monero_v8, cryptonight_monero, 8u}, nullptr },
 	{ "ryo",                 {cryptonight_heavy, cryptonight_heavy, 0u},          {cryptonight_heavy, cryptonight_heavy, 0u},   nullptr },
 	{ "stellite",            {cryptonight_monero_v8, cryptonight_stellite, 255u}, {cryptonight_monero_v8, cryptonight_monero_v8, 0u}, nullptr },
-	{ "turtlecoin",          {cryptonight_aeon, cryptonight_aeon, 0u},            {cryptonight_aeon, cryptonight_aeon, 0u},     nullptr }
+	{ "turtlecoin",          {cryptonight_aeon, cryptonight_aeon, 0u},            {cryptonight_aeon, cryptonight_aeon, 0u},     nullptr },
+    { "zelerius",            {cryptonight_monero_v8, cryptonight_monero, 6u},     {cryptonight_monero_v8, cryptonight_monero, 6u}, "pool.zelerius.com:3333" }
```
Comment (psychocrypt) [inline xmrstak/jconf.cpp]: Could you set the dev pool to `{ "zelerius",            {cryptonight_monero_v8, cryptonight_monero, 6u},     {cryptonight_monero_v8, cryptonight_monero_v8, 0u}, "pool.zelerius.com:3333" }`

Comment (zelerius) [reply xmrstak/jconf.cpp]: Yes, changed.

**Comment by fireice-uk** [comment]
Firstly please link a major pool on your network to show evidence of sustained 2MH/s hashrate.

Secondly please clean up your pr, there should be only one commit in it.

**Comment by zelerius** [comment]
PR cleaned, one commit -> [Add zelerius as a config option c931d7e](https://github.com/fireice-uk/xmr-stak/pull/2090/commits/c931d7ea333547c1ceb54bd48b56e6022c3a91e7)
These are two major pools: [pool.zelerius.org](https://pool.zelerius.org) and [pool.zelerius.ru](http://pool.zelerius.org) On the other hand this is a central pool list [miningpoolstats](https://miningpoolstats.stream/zelerius). We indicate pool.zelerius.org in the PR  because it is hosted in the official domain.

**Comment by zelerius** [comment]
I close this PR and reopen

**Pull request closed without merging**